CommonTools/PileupAlgos/plugins/PuppiProducer.cc

```diff
@@ -355,12 +353,10 @@ void PuppiProducer::fillDescriptions(edm::ConfigurationDescriptions& description
   desc.add<bool>("UseFromPVLooseTight", false);
   desc.add<bool>("UseDeltaZCut", true);
   desc.add<double>("DeltaZCut", 0.3);
-  desc.add<double>("PtMaxCharged", 0.);
-  desc.add<double>("EtaMaxCharged", 0.);
+  desc.add<double>("PtMaxCharged", 7000.);
+  desc.add<double>("EtaMaxCharged", 10.);
```

Comment:
That's in principle fine.
Perhaps, to make clear to everyone reading the config that you want to keep "all eta's" you should rather use an obviously large number, like (e.g.) "99999'"